google/ggrc-core
# GGRC-6801 Related information popup now shows mapped object type

# Issue description

Related information popup shows Control instead of Access group.

# Steps to test the changes
Login to GGRC application:
1. Open any audit page
2. Map control and access group to audit
3. Create assessment with Control type
4. Open Assessment info panel> Other attributes tab
5. Click Map icon for Related Information and map Access group snapshot to assessment
6. Click on Access group snapshot to open popup
Actual Result: Related information popup shows Control instead of Access group
Expected Result: Related information popup should show mapped object type
7. Look at the DESCRIPTION

# Solution description

Code changes done to make sure related information popup shows the mapped object type.

# Sanity checklist

- [] I have applied the correct milestone and labels.



# PR Review checklist
